common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -4407,4 +4417,13 @@ not_raising_undead_flag_trigger = {
 		text = raising_undead_flag_trigger_tooltip
 		NOT = { has_character_flag = raising_undead_flag }
 	}
+}
+
+
+#Harpy Trait
+is_possible_harpy_victim_trigger = {
+	is_female = no 
+	is_ruler = no
+	NOT = { trait = creature_dragon }
+	can_has_children_trigger = yes
 }
```

Comment:
This trigger isn't needed because `can_has_sexual_relation_with_prev` determines can harpy do IT with PREV or not.